syntax = "proto3";

message Example {
  int32 field1 = 1;       
  double field2 = 2;  
  string field3 = 3;    
}